_posts/2021-09-29-solidity-0.8.9-release-announcement.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.9 Release Announcement'
+date: '2021-09-29'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) fixes two important bugs:
+
+ - one related to [immutables of signed integer types shorter than 256 bits](https://blog.soliditylang.org/2021/09/29/signed-immutables-bug/) introduced in Solidity 0.6.5
+ - one introduced with [user defined value types](https://blog.soliditylang.org/2021/09/29/user-defined-value-types-bug/) in Solidity v0.8.8 (released two days ago).
```

Comment:
```suggestion
 - a bug introduced with [user defined value types](https://blog.soliditylang.org/2021/09/29/user-defined-value-types-bug/) in Solidity v0.8.8 (released two days ago), which makes the new feature unnecessarily gas wasteful and may create issues for tooling or contract upgrades.
```
Reading "one" and the sentence structure here felt weird to me but it's just a style question, not super important.